plugin/litedfm.vim

```diff
@@ -109,6 +105,7 @@ function! LiteDFM()
   let &ruler = get(g:, 'lite_dfm_keep_ruler', 0)
   set number
   set laststatus=0
+  set showtabline=1
```

Comment:
Before I merge, I just want to make sure this is correct. You sure this shouldn't say `set showtabline=0` here? The goal is to hide things in this function.

Reply:
You're right. I'll change that real quick.
```suggestion
  set showtabline=0
```